modules/ROOT/pages/_partials/sync/getting-started-binding-keycloak.adoc

```diff
@@ -19,7 +19,7 @@ The following command will create a secret called `sync-keycloak-doc`. It will c
 
 [source, bash]  
 ----
-oc create secret generic {sync-service}-keycloak-doc \
+oc create secret generic sync-keycloak-doc \
```

Comment:
fyi: you could have left this, but code examples require following to use attributes, see http://mrhaki.blogspot.com/2014/05/awesome-asciidoc-substitute-attribute.html